docs/dcp2_system_design.rst

```diff
@@ -960,221 +960,91 @@ Project-level matrices
-list view and the project details view.
+DUO (Data Use Ontology) codes will not be included at this time.
 
-The set of properties the metadata schema currently allows for in
-``supplementary_file`` entities is limited, so the challenge is
-how to encode matrix stratification. The schema defines the following
-properties:
 
--  ``supplementary_file.provenance.submitter_id``
 
--  ``supplementary_file.file_core.format``
+``Schema changes for CGMs (dcp3)``
 
--  ``supplementary_file.file_core.content_description``
+For release 3 and onwards, we intend to store matrices provided by data contributors (as opposed to the matrices generated by the DCP) in the informationally rich metadata schema. The CGMs will be described as ‘analysis_file’ entities, similar to DCP-generated matrices, and will be linked via a process to:
+the ‘sequence_file’ OR ‘cell suspension’ entities that contributed to it
+the ‘analysis_protocol’ that generated the file
+the ‘library_preparation_protocol’ applied to the cell suspensions that generated it.
+The CGM files will be exported from Ingest as analysis files, and imported and indexed as analysis files.
 
--  ``supplementary_file.file_core.file_name``
+Four optional fields are added to the schema to facilitate storing CGMs metadata.
 
--  ``supplementary_file.file_description``
+- “analysis_file.file_core.file_source”
+    - optional field set to one of two values, contributor-generated or DCP-generated.
 
-The ``submitter_id`` field is set to one out of a few statically
-defined v5 UUIDs that designate the matrix file as having been generated
-by a contributor. The namespace for the v5 UUID is
-``382415e5-67a6-49be-8f3c-aaaa707d82db`` and the name is one of
-the values from the tracking spreadsheet column named
-``file_source`` e.g. ``contributor``, ``hca release``,
-``arrayexpress`` and so on. A complete list can
-be found in the `Azul source`_ [9]_.
+- “analysis_file.matrix.cell_count”
+    - Optional field storing the number of cells which are analyzed in the analysis file
 
-.. _Azul source: https://github.com/DataBiosphere/azul/blob/a820e259ba9e37a94b5788a257d4c6f43fe31801/src/azul/plugins/metadata/hca/transform.py#L271
+- “analysis_protocol.matrix.data_normalization”
+    - Optional descriptive field describing the processes by which the analysis data is normalized
+    - Ideally ontologised, but as there is no appropriate ontology yet we are including it as a first step, and will be re-assessed
 
-The ``format`` property of a contributor-generated matrix file
-is set to the extension of the matrix file e.g., ``zip``,
-``csv``, ``h5ad``, ``mtx.gz``, ``tar``, ``xls`` and so on.
+- “analysis_protocol.matrix.derivation_process”
+    - Optional descriptive field describing the computational tools used to generate the analysis file
+    - Ideally ontologised, but as there is no appropriate ontology yet we are including it as a first step, and will be re-assessed
```

Comment:
Duplicative and largely redundant. Please move the "enum vs ontology" consideration into a non-normative paragraph below.